themes/powerline-multiline/powerline-multiline.theme.bash

```diff
@@ -153,7 +153,7 @@ function __powerline_battery_prompt {
     else
       color="${BATTERY_STATUS_THEME_PROMPT_GOOD_COLOR}"
     fi
-    [[ "$(ac_adapter_connected)" ]] && battery_status="${BATTERY_AC_CHAR}${battery_status}"
+    ac_adapter_connected && battery_status="${BATTERY_AC_CHAR} ${battery_status}"
```

Comment:
Without the space, the symbol was partly overlapping the numbers for me. But you're right, one could simply change the `BATTERY_AC_CHAR` variable and add the space there. I'll change that...